lib/mayaUsd/ufe/Utils.h

```diff
@@ -56,6 +56,14 @@ bool isRootChild(const Ufe::Path& path);
 MAYAUSD_CORE_PUBLIC
 SdfLayerHandle defPrimSpecLayer(const UsdPrim& prim);
 
+//! Check if the target layer has any opinions that affects a particular prim
+MAYAUSD_CORE_PUBLIC
+bool isTargetLayerHaveOpinion(const UsdPrim& prim);
+
+//! Return the target layer that has any opinions on a particular prim
+MAYAUSD_CORE_PUBLIC
+SdfLayerHandle targetLayerWithOpion(const UsdPrim& prim);
+
 MAYAUSD_CORE_PUBLIC
```

Comment:
defPrimSpecLayer, isTargetLayerHaveOpinion, targetLayerWithOpion functions don't really operate on UFE and they all only rely on USD functionalities.

I am thinking if `maya-usd\lib\usd\utils` is a better place to house such functionalities.

Reply:
Agreed, it has no dependency on UFE, nor Maya.

Reply:
That's very true, but then to be consistent you'd need to make this change to other functions in the file.  There are other utility functions in this file that are Maya-only and don't depend on UFE, so it makes you wonder where those should go --- strictly speaking not in the ufe directory.

Reply:
See 54166d2b6
@ppt-adsk  correct, for the sake of making this PR short I only moved out the USD-only utility functions from lib/maya-usd to lib/usd.